LICENSE.txt

```diff
@@ -1,28 +1,29 @@
-﻿Copyright (c) 2015-2017, Anaconda, Inc. and contributors
+﻿BSD 3-Clause License
+
+Copyright (c) 2015-2017, Anaconda, Inc. and contributors
```

Comment:
I _think_ we can also drop the `-2017` part of the copyright as we only need to specify when copyright was established (though I'm clearly not an expert on this topic). Another option would be to keep the current format but update to `2015-2022`